vladiate/vlad.py

```diff
@@ -10,20 +10,25 @@ class Vlad(object):
     def __init__(
         self,
         source,
+        fieldnames=[],
         validators={},
         default_validator=EmptyValidator,
         delimiter=None,
         ignore_missing_validators=False,
+        console_log=False,
```

Comment:
Defaulting this to `False` would be a breaking change here, as most users would be expecting the output.

I also think a better name for this would be `quiet`.

Reply:
It does not break default behavior and still returns "passed" or "failed".  The reason is this can be used as a command line validator tool and also as an API that can be integrate with an existing workflow. In the later case, it should be behave and return passed or failed status. The verbose is good for a tool use case. I will rename it "quiet"